libyul/ObjectParser.cpp

```diff
@@ -109,10 +109,8 @@ shared_ptr<Object> ObjectParser::parseObject(Object* _containingObject)
 
 shared_ptr<Block> ObjectParser::parseCode(optional<SourceNameMap> _sourceNames)
 {
-	if (currentToken() != Token::Identifier || currentLiteral() != "code")
-		fatalParserError(4846_error, "Expected keyword \"code\".");
+	expectToken(Token::Code, false);
 	advance();
```

Comment:
Isn't both of this just
```suggestion
	expectToken(Token::Code);
```
?